tests/test_globals.hpp

```diff
@@ -31,10 +31,11 @@ struct print_log_value<glm::vec3> {
         s << glm::to_string(v);
     }
 };
+BOOST_NS_MAGIC_CLOSING
 }
 }
-BOOST_NS_MAGIC_CLOSING
 
+#if BOOST_VERSION < 106400
```

Comment:
We should add an issue about this after merge so we can remove such checks in a couple of years when all distros upgraded boost.